src/pages-and-resources/PagesAndResources.jsx

```diff
@@ -12,40 +12,60 @@ import DiscussionsSettings from './discussions';
 import PageGrid from './pages/PageGrid';
 import ResourceList from './resources/ResourcesList';
 
-import { fetchPages } from './data/thunks';
+import { fetchCourseApps } from './data/thunks';
 import { useModels } from '../generic/model-store';
 import PagesAndResourcesProvider from './PagesAndResourcesProvider';
 
 function PagesAndResources({ courseId, intl }) {
-  const { path } = useRouteMatch();
+  const { path, url } = useRouteMatch();
 
   const dispatch = useDispatch();
   useEffect(() => {
-    dispatch(fetchPages(courseId));
+    dispatch(fetchCourseApps(courseId));
   }, [courseId]);
 
-  const pageIds = useSelector(state => state.pagesAndResources.pageIds);
-  const pages = useModels('pages', pageIds);
+  const courseAppIds = useSelector(state => state.pagesAndResources.courseAppIds);
+  // Each page here is driven by a course app
+  const pages = useModels('courseApps', courseAppIds);
 
   return (
     <PagesAndResourcesProvider courseId={courseId}>
-      <main>
-        <div className="container-fluid pb-3">
-          <div className="d-flex justify-content-between align-items-center border-bottom">
-            <h1 className="mt-3">{intl.formatMessage(messages.heading)}</h1>
-          </div>
-          <PageGrid pages={pages} />
-          <ResourceList />
-        </div>
+      <main className="container container-mw-lg">
+        <h3 className="mt-3">{intl.formatMessage(messages.heading)}</h3>
+        <PageGrid pages={pages} />
+        <ResourceList />
         <Switch>
           <PageRoute
             path={[
-              `${path}/discussions/configure/:appId`,
```

Comment:
Why the routes have changed? Was there a requirement to change them?

Reply:
The URL paths are build based on the app id returned by the server. The id the server is returning is `discussion` not `discussions`. We can either fix it there or here.

Reply:
makes sense 👍